tech.feliciano.pocket-casts.yml

```diff
@@ -34,7 +34,7 @@ modules:
   - name: pocket-casts
     buildsystem: simple
     build-options:
-      append-path: /usr/lib/sdk/node14/bin
+      append-path: /usr/lib/sdk/node18/bin
       env:
         CHROMEDRIVER_SKIP_DOWNLOAD: 'true'
         npm_config_nodedir: '/usr/lib/sdk/node14'
```

Comment:
Does this path need to change to `node18` also?

Reply:
App still seems to run. I've opened a new issue about it, and I'll take a look at it at the same time as I'm updating the app, when I get to it.

Reply:
Thanks for taking a look! Just noticed that when I checked the changes and thought to check